src/main/resources/io/jenkins/plugins/designlibrary/Layouts/index.jelly

```diff
@@ -0,0 +1,144 @@
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__content">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+          <div class="jdl-layout-preview__line" />
+        </div>
+        <div class="jdl-layout-preview__cards">
+          <div class="jdl-layout-preview__header" />
+          <div class="jdl-layout-preview__button" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+          <div class="jdl-layout-preview__card" />
+        </div>
+      </div>
+    </div>
+
+    <div class="jenkins-tab-pane">
+      <h2 class="jenkins-tab-pane__title">Two column</h2>
+
+      <p class="jdl-leading-paragraph">
+        Two column layouts are for 'hub' style pages, pages that have child pages
+        for the user to navigate to.
+      </p>
+
+      <p class="jdl-paragraph">Examples of such pages include 'Dashboard', 'Project' and 'Plugin Manager'.</p>
+
+      <hr />
+
+      <table class="jdl-dos-donts">
+        <thead>
+          <tr>
+            <th class="jenkins-!-color-success-color">${%do}</th>
+            <th class="jenkins-!-color-error-color">${%dont}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>${%two-column.do.1}</td>
+            <td>${%two-column.dont.1}</td>
+          </tr>
+          <tr>
+            <td>${%two-column.do.2}</td>
+            <td>${%two-column.dont.2}</td>
+          </tr>
+          <tr>
+            <td>${%two-column.do.3}</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <hr />
+
+      <h3 class="jdl-heading">${%Usage}</h3>
+
+      <p class="jdl-paragraph">The 'two-column' layout is default, so you don't need to do anything to use it.</p>
+
+      <s:code file="twoColumn.jelly" language="xml" previewable="true" />
+
+      <p class="jdl-paragraph">
+        The example above is all you need to set up a two-column page,
+        it's complete with headings, sidebar items and content.
+      </p>
+    </div>
+
+    <div class="jenkins-tab-pane">
+      <h2 class="jenkins-tab-pane__title">One column</h2>
+
+      <p class="jdl-leading-paragraph">
+        One column layouts are for pages that don't need navigation; they're ideal for standalone pages
+        that don't have child navigation.
```

Comment:
Does that mean, that our goal is to remove the sidebar for views that do not need a navigation?

Reply:
Or do those views show the old sidebar?

Reply:
Can you add a reference to an example?

Reply:
Yep, pages that have empty sidebars/nothing in them should have their sidebars removed and the layout type should be set to 'one-column'.

Reply:
is there an example of a page that looks good as one-column? I found the one-column ones feeling a bit off 
 in https://github.com/jenkinsci/jenkins/pull/6907 but it could just be because I'm used to them

Reply:
I've updated [#6907](https://github.com/jenkinsci/jenkins/pull/6907) to centre one-column layouts now as well as to increase their max-width, which I think helps make the layout easier to use + brings back the familiarity of the form not being on the far left of your screen. 

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/43062514/180816180-30cfffde-b402-4d2d-bdd5-96be115b1787.png">

I think it's an improvement (but there's definitely still more to do with layouts etc), let me know what you think 👍

Reply:
that looks better from the screenshot, I'll put it later on and try all the pages out for feeling